src/Countries/Pakistan.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Pakistan extends Country
+{
+    public function countryCode(): string
+    {
+        return 'pk';
+    }
+
+    /**
+     * @return array<string, CarbonImmutable>
```

Comment:
No need for the return type or the @link type

Reply:
Removed the doc block.